playwright/drawer-panel-dropdown.spec.ts

```
@@ -0,0 +1,334 @@
+      '(1)'
+    );
+    await actionsToggle.click();
+
+    // Select all
+    await drawerHelpers.bulkSelectAll(page);
+
+    // Verify count matches total
+    await actionsToggle.click();
+    await expect(dropdown.getByRole('menuitem', { name: /Mark selected .* as read/ })).toContainText(
+      `(${count})`
+    );
```

ESLint

[error] Replace `dropdown.getByRole('menuitem',·{·name:·/Mark·selected·.*·as·read/·})).toContainText(⏎······`(${count})`⏎····` with `⏎······dropdown.getByRole('menuitem',·{·name:·/Mark·selected·.*·as·read/·})⏎····).toContainText(`(${count})``

(prettier/prettier)